workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/Assembly.cs

```diff
@@ -0,0 +1,40 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Improbable.Gdk.Tools;
+
+namespace Improbable.Gdk.DeploymentManager.Commands
+{
+    public static class Assembly
+    {
+        public static WrappedTask<RedirectedProcessResult, AssemblyConfig> UploadAsync(AssemblyConfig config,
+            bool force = false)
+        {
+            var source = new CancellationTokenSource();
+            var token = source.Token;
+
+            var args = new[]
+            {
+                "cloud",
+                "upload",
+                config.AssemblyName,
+                "--project_name",
+                config.ProjectName,
+                force ? "--json_output --force" : "--json_output"
```

Comment:
This reads a bit oddly, bunching up two separate arguments like this

Reply:
I agree - sadly there's not an ergonomic way to conditionally add an element to an array (in the initializer). Will convert to a list